components/molecules/TreemapPrototype/special-node.tsx

```diff
@@ -26,6 +26,7 @@ const NonMemoizedSpecialNode = <Datum extends object>({
         borderWidth,
         borderColor: node.borderColor,
       }}
+      onClick={node.onClick}
```

Comment:
I moved the click here to drill down to the next level with the keyboard. The event probably bubbles down, but it makes more sense that the click is at the top level.